packages/flutter/lib/src/scheduler/scheduler.dart

```diff
@@ -84,102 +84,138 @@ class Priority {
 class Scheduler {
   /// Requires clients to use the [scheduler] singleton
   Scheduler._() {
-    ui.window.onBeginFrame = beginFrame;
+    ui.window.onBeginFrame = handleBeginFrame;
   }
 
   SchedulingStrategy schedulingStrategy = new DefaultSchedulingStrategy();
 
-  final PriorityQueue _queue = new HeapPriorityQueue<_SchedulerEntry>(
-    (_SchedulerEntry e1, _SchedulerEntry e2) {
+  final PriorityQueue _taskQueue = new HeapPriorityQueue<_TaskEntry>(
+    (_TaskEntry e1, _TaskEntry e2) {
       // Note that we inverse the priority.
       return -e1.priority.compareTo(e2.priority);
     }
   );
 
-  /// Wether this scheduler already requested to be woken up as soon as
-  /// possible.
-  bool _wakingNow = false;
+  /// Wether this scheduler already requested to be called from the event loop.
+  bool _hasRegisteredEventLoopCallback = false;
```

Comment:
If I understand the code correctly, I think this should be hasRequestedEventLoopCallback or hasScheduledEventLoopCallback. Registering the callback is what you do when you set the onFoo field to the handleFoo method. Requesting or scheduling one is what you do when you tell the other side to use the callback.